dist/common/scripts/scylla_io_setup

```diff
@@ -91,12 +91,15 @@ if __name__ == "__main__":
             else:
                 conf_dir = etcdir() + "/scylla"
             cfg = yaml.safe_load(open(os.path.join(conf_dir, "scylla.yaml")))
+
+            if 'workdir' not in cfg or not cfg['workdir']:
+                default_path = datadir()
+            else:
+                default_path = cfg['workdir']
```

Comment:
This can be rewritten in one line.

    default_path = cfg.get('workdir', '') or datadir()

Reply:
Thanks, a simple one:

```
default_path = cfg.get('workdir') or datadir()
```